strategy.md

```diff
@@ -0,0 +1,70 @@
+# Topics
+
+- Git/Github/Version Control
+
+- Webstorm/IDE/text editor
+
+- Basic Webdev (html, css, front JS)
+
+- Intermediate Webdev (html template builder, sass, JS front-end libraries)
+
+- Advanced Webdev (environments like nodejs, optimization/best practices)
+
+- Super Advanced Webdev (Webpack, Production/Hosting, Managing DevOps)
+
+- Terminal (Unix for everyone which overlaps with some of Webdev, Powershell if only available)
+
+- Documentation (Markdown, comment standards, self documenting code)
+
+# New User Issues
+Ranked in order of what I think are the most problematic, some may need to be reordered
+
+1. Have difficulties reading, having trouble understanding written instructions
+
+1. Don't know where to start looking for a solution to a problem, may not even understand the problem
+
+1. Fear of the unknown and generally with quit instead of trying to solve a problem
+
+1. Low drive for learning web development or other types of development 
+
+1. Lack of basic experience, which makes training much slower epically combined with previous points
+
+1. Large spread of different OS and hardware which makes it hard to assist and recommend solutions for new members
+
+1. Unwillingness to put in time outside of meetings which makes learning slower
+
+1. Lack of engagement on Slack
+
+# What User want to go/do
+
+- Most people seem to want to go into a some type of software industry job (would need to survey)
+
+- Members seem to mostly want to work on UX Coin or our site, although overall excitement for webdev seems low. Maybe because we haven't really started 
+
+- Unifying factor is everyone wants to go into a lucrative job/do well paying work
+
+- A couple people are more interested in the business aspect 
+
+# Solutions
+
+- More pictures in documentation. Limit screenshots of words or else it is the same problem
+
+- Not sure of a solution for problem #2. Docs should aim to solve this problem somehow by increasing discoverability in a non-threatening way
+
+- Make intro as gentle as possible. May want to consider dropping Github and just having members editing html-starter without version control.
+
+- Use GUI as much as possible, have good documentation for it
+
+- Document the extreme basics of webdev, make it as easy to refer back to as possibly by allowing users to bookmarks documentation pages internally
+
+- Make documentation as hype as possible, act exited
+
+- Have a branching path solution for different Operating Systems, like a flow chart
```

Comment:
https://github.com/kamranahmedse/developer-roadmap

You can use this as a reference. Even show people this flowchart as a guideline we would need to follow as opposed to making our own.